src/2XXXR0_a_motivation_scope_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -121,9 +121,8 @@ It is no longer only the space industry or experienced pilots that benefit from
 operations of some machines. We live in a world where more and more ordinary people trust
 machines with their lives daily. In the near future, we will be allowed to sleep while our
 car autonomously drives us home from a late party. As a result, much more C++ engineers are
-expected to write life-critical software today than it was a few years ago. Unfortunately,
-there is insufficient training and experience in that domain, and the C++ language does not
-change fast enough to enforce safe-by-construction code.
+expected to write life-critical software today than it was a few years ago. However writing safety-ciritical code requires extensive training and experiece, both which are in short demand. While there exists some standards and guidelines such as MISRA C++ [@MISRA_CPP] with the aim of enforcing the creation of safe code in C++, they are cumbersome to use and tend to shift the burden on the discipline of the programmers to enforce these. At the time of writing the C++ language does not change fast enough to enforce safe-by-construction code.
+
```

Comment:
As you know, I am not a fan of it. I probably read too many books and read too many articles online where text is justified from left to right instead of breaking it with a new line after every thought. Seeing a column of text with so different line lengths, at least for me, makes it harder to read and makes my eyes tired.

Reply:
I try to keep the column of text around 100 characters wide and break each new thought with an empty line and that works better for me.

Reply:
All of the links you provided assume that readers use only the generated version of a document and to not read the source file. For example:

> For a reader, semantic line breaks are entirely invisible — no changes to the source text appear in the final rendered output.

> I ask students whether or not the Sphinx text files in their project will be read by end-users. If not, then I encourage students to treat the files as private “source code” that they are free to format semantically.

However, I think of a Markdown as a special format [that is easy to read in its source code form](https://en.wikipedia.org/wiki/Markdown). I often do not bother to generate a document from the *.md and read the source as it is. I find the files using Semantic Line Breaks to be much harder to read. However, I see the benefits of sembr when I do not have to reformat text to the desired column width after inserting or removing a few words within a sentence. There is always some compromise to be made...